sdk/src/types/block/semantic/mod.rs

```diff
@@ -262,7 +339,7 @@ impl<'a> SemanticValidationContext<'a> {
             self.output_mana = self.output_mana.checked_add(mana).ok_or(Error::CreatedManaOverflow)?;
 
             // Add allotted mana
-            for mana_allotment in self.transaction.allotments() {
+            for mana_allotment in (*self.transaction.allotments()).iter() {
```

Comment:
can't just be `self.transaction.allotments().iter()`?

Reply:
Ah yeah sure, thanks. A remnant from before I changed the function I guess.

Reply:
Changed